### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SuppDoc**
Add accepted paper in TIP2021

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1801,7 +1801,9 @@ arXiv, 2019. [PDF](https://arxiv.org/pdf/1904.04805), [Video](https://neurorobot
 - <a name="Gehrig20cvpr"></a>Gehrig, D., Gehrig, M., Hidalgo-Carrió, J., Scaramuzza, D.,  
 *[Video to Events: Recycling Video Datasets for Event Cameras](http://rpg.ifi.uzh.ch/docs/CVPR20_Gehrig.pdf)*,  
 IEEE Conf. Computer Vision and Pattern Recognition (CVPR), 2020. [YouTube](https://youtu.be/uX6XknBGg0w),  [Project page](https://github.com/uzh-rpg/rpg_vid2e)
-
+- <a name="Deng21eventkd"></a>Y. Deng, H. Chen, H. Chen and Y. Li,
```
Comment (guillermogb) [inline README.md:1804]: Hello.
Could you please follow the [convention](https://github.com/uzh-rpg/event-based_vision_resources/blob/master/Contributing.md)?
key: Deng21eventkd -> Deng21tip (unless it is already taken)
Names: Y. Deng, H. Chen, ... -> the other way around: Deng, Y., Cheng, H., ...
IEEE Transactions on Image Processing -> IEEE Trans. Image Process.  (IEEE abbreviation).

**Review by guillermogb — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (SuppDoc) [reply README.md]: Of course, thanks for your notice

File: README.md [hunk 2]
```diff
@@ -1801,7 +1801,9 @@ arXiv, 2019. [PDF](https://arxiv.org/pdf/1904.04805), [Video](https://neurorobot
 - <a name="Gehrig20cvpr"></a>Gehrig, D., Gehrig, M., Hidalgo-Carrió, J., Scaramuzza, D.,  
 *[Video to Events: Recycling Video Datasets for Event Cameras](http://rpg.ifi.uzh.ch/docs/CVPR20_Gehrig.pdf)*,  
 IEEE Conf. Computer Vision and Pattern Recognition (CVPR), 2020. [YouTube](https://youtu.be/uX6XknBGg0w),  [Project page](https://github.com/uzh-rpg/rpg_vid2e)
-
+- <a name="Deng21tip"></a> Deng, Y., Chen, H., Chen, H. and Li, Y.
```
Comment (guillermogb) [inline README.md:1804]: We are almost there... two missing blank spaces are needed to start a new line after the author names:

image

**Review by guillermogb — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (SuppDoc) [reply README.md]: Sorry for taking up so much time, it should be fine this time.

**Review by guillermogb — APPROVED** [review]
Thank you.

**Pull request merged**